akka-actor-typed/src/main/scala/akka/actor/typed/internal/routing/PoolRouterImpl.scala

```diff
@@ -42,17 +42,17 @@ private final class PoolRouterImpl[T](
     poolSize: Int,
     behaviorFactory: () => Behavior[T],
```

Comment:
Does this mean we could go back to only taking a `Behavior` here instead of a factory? Feels a bit strange that we have two different solutions to the same problem across the API.

Reply:
Created #27807 to track

Reply:
yes, I think that would be safe